buildkite/common

```diff
@@ -22,9 +22,15 @@ GIT_BRANCH=$(echo $GIT_BRANCH | sed 's;/;-;g')
 
 BUILDKITE_DOCKER_AUTH_PATH=/var/lib/buildkite-agent/.docker/config.json
 
+if [ -n "${ORDERLY_SERVER_VERSION-}" ]; then
```

Comment:
Normally  we do

```
ROOT=$(realpath $HERE/..)
```

at the top of the common script to isolate the existing global envvar (though here you'll need lower case it seems). With that, then your line below is `MONTAGU_ORDERLY_SERVER_VERSION=$(<$ROOT/config/orderly_server_version)`